README.md

```diff
@@ -106,6 +106,12 @@ confetti({
 });
 ```
 
+In some cases you may need to destoy confetti, you can do this by calling method `destroy`
+
+```
+confetti.destroy();
```

Comment:
Please include this in the API section rather than in the middle of the example prose.